electrum/gui/qt/fee_slider.py

```diff
@@ -5,16 +5,16 @@
 from PyQt6.QtWidgets import QSlider, QToolTip, QComboBox
 
 from electrum.i18n import _
+from electrum.fee_policy import FeeMethod
 
 
 class FeeComboBox(QComboBox):
 
     def __init__(self, fee_slider):
         QComboBox.__init__(self)
-        self.config = fee_slider.config
         self.fee_slider = fee_slider
         self.addItems([_('Static'), _('ETA'), _('Mempool')])
-        self.setCurrentIndex((2 if self.config.use_mempool_fees() else 1) if self.config.is_dynfee() else 0)
+        self.setCurrentIndex(int(self.fee_slider.fee_policy.method) - 1)
```

Comment:
ah we have the same implicit magic mapping here too :/

Reply:
you've changed this in https://github.com/spesmilo/electrum/pull/9597/commits/b4e95f7d7a4c2c7b43f67b8ee53c617cc4094c7b, but it is still not fixed. The whole thing still implicitly depends on the order of items in the `addItems` call:
```
self.addItems([_('Static'), _('ETA'), _('Mempool')])
```
See e.g.:
https://github.com/spesmilo/electrum/commit/ccc1897f36b3f8017b669dd15e38c96534a6d524

Reply:
now uses `self.addItems([x.name_for_GUI() for x in FeeMethod.slider_values()])`